packages/react-native-win32/src/Libraries/Components/StatusBar/StatusBar.win32.ts

```diff
@@ -1,5 +1,10 @@
 /**
- * This copies what StatusBar.android.js does
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
The comment on the left isn't accurate.

An aside, we're missing copyright headers on a lot of these files. From previous discussions it seems like that isn't a major deal, but we should probably address that.

Reply:
If we want to release it in Open Source, full header should be:

// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

---
In reply to: [385375547](https://github.com/microsoft/react-native-windows/pull/4205#discussion_r385375547) [](ancestors = 385375547)